packages/aptos-nextra-components/src/utils/cn.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
```

Comment:
What's cn?  Doc comment?

Reply:
classnames merge function @blakezimmerman made

Reply:
It's worth noting that it also de-dupes any redundant Tailwind classes.